piq/functional/filters.py

```diff
@@ -0,0 +1,27 @@
+import torch
```

Comment:
Let's add short docstring to describe the file similar to colour space conversions.